workflow/Snakefile

```diff
@@ -925,6 +925,15 @@ rule run_enrich_wg:
             {input.reference} {input.obo} {output} 2> {log}
     """
 
+rule cog_plot:
```

Comment:
Could this rule also be applied to GO terms and KEGG KOs? If so you could have three inputs and three outputs, and in the actual rule run the script three times?